x/projects/keeper/creation.go

```diff
@@ -264,6 +265,12 @@ func (k Keeper) snapshotProject(ctx sdk.Context, projectID string, block uint64)
 			utils.Attribute{Key: "project", Value: projectID},
 			utils.Attribute{Key: "block", Value: ctx.BlockHeight()},
 		)
+
+		details := map[string]string{
+			"projectID": projectID,
+			"block":     strconv.FormatInt(int64(block), 10),
+		}
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.ProjectResetFailEventName, details, "reset projects failed: unable to find project plan")
```

Comment:
fix error:
project plan -> project

Reply:
Fixed in d30c3711ed7dbfef50f7cdb3f2b39d2d54695f42